internal/handler/wallet/get_test.go

```diff
@@ -0,0 +1,121 @@
+package wallet
+
+import (
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/go-openapi/loads"
+	"github.com/massalabs/thyra-plugin-massa-wallet/api/server/models"
+	"github.com/massalabs/thyra-plugin-massa-wallet/api/server/restapi"
+	"github.com/massalabs/thyra-plugin-massa-wallet/api/server/restapi/operations"
+)
+
+func configureAPIServerGet() (*operations.MassaWalletAPI, error) {
+	swaggerSpec, err := loads.Analyzed(restapi.SwaggerJSON, "")
+	if err != nil {
+		return nil, err
+	}
+
+	localAPI := operations.NewMassaWalletAPI(swaggerSpec)
+	server := restapi.NewServer(localAPI)
+
+	var walletStorage sync.Map
+
+	localAPI.RestWalletListHandler = NewList(&walletStorage)
+
+	server.ConfigureAPI()
+
+	return localAPI, nil
+}
+
+func Test_walletGet_Handle(t *testing.T) {
+	api_get, err := configureAPIServerGet()
+	if err != nil {
+		panic(err)
+	}
+
+	type want struct {
+		header     http.Header
+		statusCode int
+	}
+
+	testsGet := []struct {
+		name     string
+		nickname string
+		want     want
+	}{
+		{"Passed_list_empty", ``, want{header: http.Header{"Content-Type": {"application/json"}}, statusCode: 200}},
+		{"Passed_list_with_wallets", "precondition_wallet", want{header: http.Header{"Content-Type": {"application/json"}}, statusCode: 200}},
+	}
+
+	t.Run(testsGet[0].name, func(t *testing.T) {
```

Comment:
Why don't you use a for loop ? Here you are only using the first element of teh array.

Reply:
I'm not sure you should use test table here if what you are doing is to do two different test.
In doubt, you should prefere simplicity